flink-contrib/docker-flink/docker-compose.yml

```diff
@@ -16,21 +16,22 @@
 # limitations under the License.
 ################################################################################
 
-version: "2"
+# Set the FLINK_DOCKER_IMAGE_NAME environment variable to override the image name to use
+
+version: "2.1"
 services:
   jobmanager:
-    image: flink
-    container_name: "jobmanager"
+    image: ${FLINK_DOCKER_IMAGE_NAME:-flink}
     expose:
       - "6123"
```

Comment:
the variable in yml doesn't depends variable from Dockerfile.
see https://docs.docker.com/compose/compose-file/#variable-substitution